robottelo/products.py

```diff
@@ -763,7 +763,14 @@ def need_subscription(self):  # type: () -> bool
     def organization(self):
         return self._org
 
-    def add_item(self, item):  # type: (BaseRepository) -> None
+    def add_item(self, item) -> None:
+        """
+        Add repository to collection
+
+        :type item: BaseRepository
+        :param item: Item to add
```

Comment:
Yes, that's correct.

`:type item: BaseRepository` is for MyPy/IDE to tell you what is the type of argument.

`:param item:` is for Sphinx to tell you in docs what arguments are allowed.